src/gui/torrentmodel_p.cpp

```diff
@@ -0,0 +1,325 @@
+#include "./torrentmodel_p.h"
```

Comment:
Move this include at the rest of project includes (after utils/colorutils.h).
Don't mix system/Qt/project includes.